core/src/main/java/greencity/controller/ManagementEmployeeController.java

```diff
@@ -138,6 +138,27 @@ public ResponseEntity<HttpStatus> deleteEmployee(@PathVariable Long id) {
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
+    /**
+     * Controller activate employee.
+     *
+     * @return {@link HttpStatus}
+     * @author Oksana Spodaryk.
+     */
+    @ApiOperation(value = "Activate employee")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @PreAuthorize("@preAuthorizer.hasAuthority('DEACTIVATE_EMPLOYEE', authentication)")
```

Comment:
why do we use `DEACTIVATE_EMPLOYEE` authority for activation?

Reply:
Because if user jave this authority he also can ACTIVATE_EMPLOYEE